src/assets/i18n/el.json5

```diff
@@ -0,0 +1,2242 @@
+	"form.search-help": "Κάντε κλικ εδώ για να αναζητήσετε μια υπάρχουσα αλληλογραφία",
+	"form.submit": "Αποθήκευση",
+	"grant-deny-request-copy.deny": "Μην στείλετε αντίγραφο",
+	"grant-deny-request-copy.email.back": "Επιστροφή",
+	"grant-deny-request-copy.email.message": "Μήνυμα",
+	"grant-deny-request-copy.email.message.empty": "Παρακαλώ εισάγετε ένα μήνυμα",
+	"grant-deny-request-copy.email.permissions.info": "Μπορείτε να χρησιμοποιήσετε αυτήν την ευκαιρία για να επανεξετάσετε τους περιορισμούς πρόσβασης στο έγγραφο, για να αποφύγετε να απαντήσετε σε αυτά τα αιτήματα. Εάν θέλετε να ζητήσετε από τους διαχειριστές του αποθετηρίου να καταργήσουν αυτούς τους περιορισμούς, επιλέξτε το παρακάτω πλαίσιο.",
+	"grant-deny-request-copy.email.permissions.label": "Αλλαγή σε ανοιχτή πρόσβαση",
+	"grant-deny-request-copy.email.send": "Στείλετε",
+	"grant-deny-request-copy.email.subject": "Θέμα",
+	"grant-deny-request-copy.email.subject.empty": "Εισαγάγετε ένα θέμα",
+	"grant-deny-request-copy.grant": "Αποστολή αντιγράφου",
+	"grant-deny-request-copy.header": "Αίτημα αντιγραφής εγγράφου",
+	"grant-deny-request-copy.home-page": "Μετάβαση στην αρχική σελίδα",
+	"grant-deny-request-copy.intro1": "Εάν είστε ένας από τους συντάκτες του εγγράφου <a href='{{ url }}'>{{ name }}</a>, χρησιμοποιήστε μία από τις παρακάτω επιλογές για να απαντήσετε στο αίτημα του χρήστη.",
+	"grant-deny-request-copy.intro2": "Αφού επιλέξετε μια επιλογή, θα σας παρουσιαστεί μια προτεινόμενη απάντηση μέσω email την οποία μπορείτε να επεξεργαστείτε.",
+	"grant-deny-request-copy.processed": "Αυτό το αίτημα έχει ήδη διεκπεραιωθεί. Μπορείτε να χρησιμοποιήσετε το παρακάτω κουμπί για να επιστρέψετε στην αρχική σελίδα.",
+	"grant-request-copy.email.message": "Αγαπητέ {{ recipientName }},\nΩς απάντηση στο αίτημά σας, έχω τη χαρά να σας στείλω επισυναπτόμενο αντίγραφο του αρχείου(ων) που αφορά το έγγραφο: \"{{ itemUrl }}\" ({{ itemName }}), του οποίου είμαι συντάκτης.\n\nΤις καλύτερες ευχές,\n{{ authorName }} <{{ authorEmail }}>",
+	"grant-request-copy.email.subject": "Ζητήστε αντίγραφο εγγράφου",
+	"grant-request-copy.error": "Παρουσιάστηκε σφάλμα",
+	"grant-request-copy.header": "Αίτημα αντιγραφής παραχώρησης εγγράφου",
+	"grant-request-copy.intro": "Αυτό το μήνυμα θα σταλεί στον αιτούντα του αιτήματος. Το(τα) έγγραφο(α) που ζητήθηκαν θα επισυναφθούν.",
+	"grant-request-copy.success": "Το αίτημα τεκμηρίου εγκρίθηκε με επιτυχία",
+	"health-page.error.msg": "Η υπηρεσία ελέγχου υγείας δεν είναι διαθέσιμη προσωρινά",
+	"health-page.heading": "Υγεία",
+	"health-page.info-tab": "Πληροφορίες",
+	"health-page.property.status": "Κωδικός κατάστασης",
+	"health-page.section-info.app.title": "Εφαρμογή Backend",
+	"health-page.section-info.java.title": "Java",
+	"health-page.section.db.title": "Βάση δεδομένων",
+	"health-page.section.geoIp.title": "GeoIp",
+	"health-page.section.no-issues": "Δεν εντοπίστηκαν προβλήματα",
+	"health-page.section.solrAuthorityCore.title": ""Solr: πυρήνας authority"",
```

Comment:
@kostisalex : This line is throwing an error for me when I attempt to load `el.json5`.  It looks like it has too many quotes (")?  I think it should say: `"Solr: πυρήνας authority"`

Reply:
@kostisalex Fixed and commited